lib/rdkafka/admin.rb

```diff
@@ -51,6 +52,16 @@ def create_topic(topic_name, partition_count, replication_factor)
         raise Rdkafka::Config::ConfigError.new(error_buffer.read_string)
       end
 
+      unless topic_config.empty?
+        topic_config.each do |key, value|
+          Rdkafka::Bindings.rd_kafka_NewTopic_set_config(
```

Comment:
Wouldn't it make things consistent to have lower case methods names?

Reply:
My understanding is that it has to match exactly what the C function is called, so I'm not sure this can be lowered. Happy to be wrong though.

https://github.com/edenhill/librdkafka/blob/master/src/rdkafka_admin.c#L1165

Other binding examples in the same situation: https://github.com/appsignal/rdkafka-ruby/blob/main/lib/rdkafka/bindings.rb#L253-L280